pkg/storage/mongodb/cslstore/csl_store.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package cslstore
+
+import (
+	"errors"
+	"fmt"
+
+	mongodbext "github.com/hyperledger/aries-framework-go-ext/component/storage/mongodb"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/trustbloc/vcs/pkg/service/credentialstatus"
+	"github.com/trustbloc/vcs/pkg/storage/mongodb"
+)
+
+const (
+	cslStoreName               = "credentialstatus"
+	latestListIDDBEntryKey     = "LatestListID"
+	mongoDBDocumentIDFieldName = "_id"
+	idFieldName                = "id"
+)
+
+// Store manages profile in mongodb.
+type Store struct {
+	mongoClient *mongodb.Client
+}
+
+type latestListIDEntry struct {
```

Comment:
We do not call things in mongodb as "Entry", but rather as "Document". It is more inline with mongo terminology. 
Rename please to latestListIDDocument or latestListIDDoc

Reply:
sure, updated